pages/geograhypage.vue

```diff
@@ -16,21 +16,25 @@ export default {
                         name: 'USA',
                         children: [
                             { name: 'CA', value: 1},
-                            { name: 'GA', value: 5},
+                            { name: 'GA', value: 3},
                         ]
                     },
                     {
                         name: 'Canada',
                         children: [
-                            { name: 'ON', value: 2},
-                            { name: 'BC', value: 3},
+                            { name: 'ON', value: 5},
+                            { name: 'BC', value: 7},
+                            { name: 'AL', value: 7}
+                        ]
+                    },
+                    {
+                        name: 'UK',
+                        children: [
+                            { name: 'England', value: 5},
```

Comment:
Instead of value, you have to have the actual users. Value is just a number here.
Please use the original data format for users, where a list of users (an array of objects) is given to you and each object is a user and has many attributes, one being the location, which itself is an object having two attributes of `country` and `state`.